test/system_test.py

```diff
@@ -141,12 +152,14 @@ def spack_env_dev_install_and_test(spack_env: str,
                    env=spack_env,
                    srun=False)
 
-    log_with_spack('spack install --test=root -n -v',
-                   'system_test',
-                   log_filename,
-                   cwd=unique_folder,
-                   env=spack_env,
-                   srun=False)
+    if not out_of_source:
+        # not all relevante files for testing are synced in spack recipe
```

Comment:
I don't understand what this comment is trying to tell me.
Is it trying to tell me, if it's an in-source build, not all relevant files are synced. Okay, but why?
Or is it an observation. Not all files are synced, therefore .... ?

Reply:
Let me clarify:
The testing infrastructure is not synced in the spack-recipe. I.e. `scripts/spack/test.py`.
Therfore we cannot run tests in an out-of-source build.

Reply:
I will make the comment more precise.

Reply:
better now?